taxonomy.xml

```diff
@@ -381,57 +381,59 @@
-             <catDesc>ʾaqʷǝyāṣāt</catDesc>
-          </category>
-          <category xml:id="Bleeding" corresp="https://betamasaheft.eu/authority-files/Bleeding/main">
-             <catDesc>Bleeding</catDesc>
-          </category>
-          <category xml:id="Malaria" corresp="https://betamasaheft.eu/authority-files/Malaria/main">
-             <catDesc>Malaria</catDesc>
-          </category>
-          <category xml:id="Epilepsy" corresp="https://betamasaheft.eu/authority-files/Epilepsy/main">
-             <catDesc>Epilepsy</catDesc>
-          </category>
-          <category xml:id="ChestPain" corresp="https://betamasaheft.eu/authority-files/ChestPain/main">
-             <catDesc>Chest Pain</catDesc>
-          </category>
-          <category xml:id="BindingDemons" corresp="https://betamasaheft.eu/authority-files/BindingDemons/main">
-             <catDesc>Binding Demons</catDesc>
-          </category>
-          <category xml:id="DrowningDemons" corresp="https://betamasaheft.eu/authority-files/DrowningDemons/main">
-             <catDesc>Drowning Demons</catDesc>
+          <category>
+             <desc>Targets</desc>
+             <category>
+                <desc>Diseases</desc>
+                <category xml:id="Disease"
+                   corresp="https://betamasaheft.eu/authority-files/Disease/main">
+                   <catDesc>Any disease </catDesc>
+                </category>
+                <category xml:id="DiseaseFever"
+                   corresp="https://betamasaheft.eu/authority-files/DiseaseFever/main">
+                   <catDesc>Any disease featuring fever as the main symptom (including
+                      malaria)</catDesc>
+                </category>
+                <category xml:id="DiseaseMental"
+                   corresp="https://betamasaheft.eu/authority-files/DiseaseMental/main">
+                   <catDesc>Any disease of mental or neurological nature, including epilepsy, fits,
+                      madness</catDesc>
+                </category>
+                <category xml:id="DiseasePain"
+                   corresp="https://betamasaheft.eu/authority-files/DiseasePain/main">
+                   <catDesc>Any disease featuring pain, including colic, rheumatic pain, headache, or
+                      chestpain</catDesc>
+                </category>
+                <category xml:id="DiseaseWomen"
+                   corresp="https://betamasaheft.eu/authority-files/DiseaseWomen/main">
+                   <catDesc>Any disease or condition connected with female health, including
+                      infertility and menstruation</catDesc>
+                </category>
+             </category>
+             <category>
+                <desc>Demonic entities</desc>                
+                <category xml:id="EvilEye"
+                   corresp="https://betamasaheft.eu/authority-files/EvilEye/main">
+                   <catDesc>Evil Eye</catDesc>
```

Comment:
Here it is probably useful to add some explanations, especially whether ʾaynat and budā shall be represented by EvilEye or not. To my understanding, both terms may fit to this category, but are quiet ambivalent in their meaning (Cf. https://github.com/BetaMasaheft/Documentation/issues/2592).